source-code/ide-extension/src/plugin/plugin.ts

```diff
@@ -0,0 +1,16 @@
+import { type IdeExtensionConfig, validateIdeExtensionConfig } from "./schema.js"
+import type { InlangConfig } from "@inlang/core/config"
+import { createPlugin } from "@inlang/core/plugin"
+
+// ------------------------------------------------------------------------------------------------
+
+export const ideExtensionPlugin = createPlugin<IdeExtensionConfig>(({ settings }) => ({
```

Comment:
hey, you told me to not use `pluginConfig` and use `settings` instead :D I would expect `IdeExtensionConfig` to be named `IdeExtensionSettings` ;)

Reply:
@samuelstroschein Done.